server/src/routines/game.ts

```diff
@@ -557,11 +555,9 @@ function* turnActionsSaga(con: GameController, turnActionChannel: any) {
 		con.game.battleLog.sendLogs()
 
 		const playerAI = getPlayerAI(con.game)
-		if (playerAI && !playerAISagaRunning) {
+		if (playerAI) {
```

Comment:
I can't figure out what the `!playerAISagaRunning` code was even there for so I removed it.

Reply:
I think this was to prevent multiple turn action sagas from starting when evilx yielded multiple actions, but I'm not sure why that would be necessary with the current system